L1Trigger/RPCTechnicalTrigger/interface/TTUTrackingAlg.h

```diff
@@ -110,14 +98,11 @@ class TTUTrackingAlg : public TTULogic {
     std::vector<Seed*> m_seeds;
     
   private:
-    
+    std::unique_ptr<Seed> m_none;
```

Comment:
Previously, if `m_tracklength == -1` it meant the destructor should delete the only entry in `m_seeds`. However, there is nothing stopping calling `add` after `addnone()`. Explicitly managing a _none_ Seed is cleaner.